closed/Intel/code/retinanet/pytorch-cpu/run_offline_accuracy.sh

```diff
@@ -14,17 +26,17 @@ fi
 
 export MALLOC_CONF="oversize_threshold:1,background_thread:true,metadata_thp:auto,dirty_decay_ms:9000000000,muzzy_decay_ms:9000000000"
 
-export LD_PRELOAD=${CONDA_PREFIX}/lib/libjemalloc.so
+#export LD_PRELOAD=${CONDA_PREFIX}/lib/libjemalloc.so
```

Comment:
good to unmark the line. it helps perf

Reply:
Just to clarify, I comment out this line because I find this line is commented out in the "aws" branch.